packages/legend-art/src/components/panel/PanelDropZone.tsx

```diff
@@ -57,9 +57,26 @@ export const PanelEntryDropZonePlaceholder: React.FC<{
   return (
     <div className={clsx(['dnd__entry-dropzone__placeholder', className])}>
       <div className="dnd__entry-dropzone__placeholder__content">
-        {label && (
-          <div className="dnd__entry-dropzone__placeholder__label">{label}</div>
-        )}
+        {label && <div> {label}</div>}
+      </div>
+    </div>
+  );
+};
+
+export const PanelEntryDropZonePlaceholderWithoutBorder: React.FC<{
```

Comment:
why don't we make this a configuration instead? Or again, let people pass in some `children` should they want to customize